hermes/probe/create_file.go

```diff
@@ -0,0 +1,124 @@
+package probe
+
+import (
+	"context"
+	"crypto/sha1"
+	"fmt"
+	"io"
+	"math/rand"
+	// "os"
+	// "sync"
+
+	// "cloud.google.com/go/storage"
+	"github.com/googleapis/google-cloud-go-testing/storage/stiface"
+	// "github.com/google/cloudprober/logger"
+	// "github.com/google/cloudprober/metrics"
+	// "github.com/google/cloudprober/probes/options"
+	// "github.com/google/cloudprober/targets/endpoint"
+	// "github.com/googleinterns/step224-2020/blob/master/hermes/hermes.go"
+)
+
+const (
+	begin = 1  // ID of the first HermesFile
+	end   = 50 // ID of the last HermesFile
+	maxFileSize = 1000 // maximum allowed file size in bytes 
+)
+
+type RandomHermesFile struct {
```

Comment:
I don't think it's necessary to include "Hermes" in the name.

Reply:
Removed